p/themes/base-theme/template.css

```diff
@@ -59,6 +59,10 @@ p {
 	font-size: 1em;
 }
 
+p.help {
+	margin: 5px 0 0.5em !important;
```

Comment:
Could you try to find a rule without `!important` ?

Reply:
I'm not good enough in CSS for that, sorry :s

It's needed because of the "prompt" class. I see that some other rules are using it, so I thought it was okay.

Reply:
You could try something like
```css
* p.help {
```

or

```css
p.help, .prompt p.help {
```

Reply:
The * p.help suggestion's a bit ugly to me. (Not objectionable like !important, just dispreferred.)

Reply:
@Alkarex So I tested:
- `* p.help {`, it doesn't work (still using prompt `p` marging)
<img width="253" alt="Screenshot 2020-04-20 at 12 59 34" src="https://user-images.githubusercontent.com/1971659/79744677-147d7f00-8307-11ea-97ba-696217c101a4.png">

- `p.help, .prompt p.help {` seems to works:
<img width="256" alt="Screenshot 2020-04-20 at 13 00 13" src="https://user-images.githubusercontent.com/1971659/79744707-219a6e00-8307-11ea-9ae8-edb6231e167c.png">

So I guess we can retain the second solution. But isn't breaking a bit encapsulation / CSS hierarchy ?

Reply:
This is because `.prompt p` was a bit too general for a rule